contracts/bridge/Bridge.sol

```diff
@@ -172,45 +172,43 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
     }
 
     // _requestKLAYTransfer requests transfer KLAY to _to on relative chain.
-    function _requestKLAYTransfer(address _to, uint256 _fee) internal {
+    function _requestKLAYTransfer(address _to, uint256 _feeLimit) internal {
         require(isRunning, "stopped bridge");
         require(msg.value > 0, "zero msg.value");
-        require(msg.value > _fee, "insufficient amount");
-        require(feeOfKLAY == _fee, "invalid fee");
+        require(msg.value > _feeLimit, "insufficient amount");
 
-        _payKLAYFee(_fee);
+        _payKLAYFeeAndRefundChange(_feeLimit);
 
         emit RequestValueTransfer(
             TokenKind.KLAY,
             msg.sender,
-            msg.value.sub(_fee),
+            msg.value.sub(_feeLimit),
             address(0),
             _to,
             requestNonce,
             "",
-            _fee
+            _feeLimit
         );
         requestNonce++;
     }
 
     // () requests transfer KLAY to msg.sender address on relative chain.
     function () external payable {
-        _requestKLAYTransfer(msg.sender, feeOfKLAY);
+        _requestKLAYTransfer(msg.sender, msg.value);
     }
 
     // requestKLAYTransfer requests transfer KLAY to _to on relative chain.
-    function requestKLAYTransfer(address _to, uint256 _fee) external payable {
-        _requestKLAYTransfer(_to, _fee);
+    function requestKLAYTransfer(address _to, uint256 _feeLimit) external payable {
+        _requestKLAYTransfer(_to, _feeLimit);
```

Comment:
Dear reviewer,
I would like to know what do you think if `_amount` is needed in this method?
`_amount` is same with `msg.value` - `_feeLimit`.
`_amount` means how much KLAY will be transfered to `_to`.

In this transaction, user writes msg.value and `_feeLimit`. So `_amount` can be calculated. 
But To user, would it be better UX to have `_amount` written.

Reply:
Then, what if real fee is smaller than _feeLimit? How do we handle remaining KLAYs?

Reply:
Is this the case used when we send some KLAYs from the service chain to the main chain?

Reply:
This code is used for both ways (SC -> MC, MC -> SC). :)

Reply:
@KimKyungup I think that `_amount` is not necessary.

Reply:
For the case of MC -> SC, wouldn't it be double paid due to the transaction fee?

Reply:
As I told you in offline. It's right. :)
Also, as we talked, I will change _feeLimit -> _amount. Thanks. @kjhman21 @aidan-kwon .